lc-hax/Scripts/Modules/ESPMod.cs

```diff
@@ -64,10 +63,11 @@ public class ESPMod : MonoBehaviour {
         HaxObjects.Instance?.EnemyAIs.ForEach(nullableEnemy => {
             if (!nullableEnemy.IsNotNull(out EnemyAI enemy)) return;
             if (enemy is DocileLocustBeesAI or DoublewingAI) return;
+            if (!enemy.skinnedMeshRenderers.FirstOrDefault().IsNotNull(out SkinnedMeshRenderer meshRenderer)) return;
 
             this.RenderBounds(
                 camera,
-                enemy.skinnedMeshRenderers[0],
+                meshRenderer,
```

Comment:
Does this somehow work better?

Reply:
yes, this fixes `IndexOutOfRangeException` log spam

Reply:
Oh that's strange. Some enemies don't have renderers? 🤔

Reply:
not sure sure why, but sometimes the array is indeed empty